release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -41,19 +41,36 @@ This release adds improvements related to the following components and concepts.
 === {op-system-first}
 
 [id="ocp-4-7-enhanced-disk-provisioning"]
-==== Enhanced disk provisioning for LUKS and RAID devices
+==== Enhanced disk provisioning for LUKS, RAID, and FBA DASD
 
-{op-system} 4.7 includes several improvements to disk provisioning for bare metal deployments:
+{product-title} 4.7 includes several improvements to disk provisioning for bare metal deployments:
 
 * Native Ignition support for LUKS disk encryption provides additional configurability for encrypted root filesystems, as well as support for encryption of additional data filesystems.
 * {op-system} now supports boot disk mirroring, except on s390x, providing redundancy in the case of disk failure.
 * {op-system} on s390x can be installed onto FBA DASD disks.
 
 [NOTE]
 ====
-LUKS configuration can no longer be performed in the `/etc/clevis.json` file. Provisioning fails if `/etc/clevis.json` is specified in the machine config.
+On new clusters, LUKS configuration must use the native Ignition mechanism, as provisioning fails if the legacy `/etc/clevis.json` file is included in the machine config. On clusters that are upgrading from {product-title} 4.6 or earlier, LUKS can only be configured by using `/etc/clevis.json`.
 ====
 
+* Multipath encryption is now available for root devices attached to clusters that are created using {product-title} 4.7 or higher. This is a useful day-2 feature that can be set up by configuring the machine config, such as shown in the following example:
++
+[source,yaml]
+--
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: "master"
+  name: 99-master-kargs-mpath
+spec:
+  kernelArguments:
+    - 'rd.multipath=default'
+    - 'root=/dev/disk/by-label/dm-mpath-root'
+--
```

Comment:
@Prashanth684 and @bgilbert PTAL based on [this comment](https://github.com/openshift/openshift-docs/pull/28368#discussion_r555050252). I don't have any other info on this feature, so please suggest revisions as necessary.

Reply:
Elsewhere in the release notes we just have brief summaries of the new functionality.  I'd suggest moving this example to the storage provisioning docs, and retaining only the first sentence as an additional bullet at the top of this section.